src/ipyaladin/aladin.py

```diff
@@ -183,6 +187,39 @@ def target(self, target: Union[str, SkyCoord]) -> None:
             }
         )
 
+    @property
+    def wcs(self) -> WCS:
+        """The WCS of the Aladin Lite widget.
+
+        You need to call synchronize_wcs in a previous
+        cell before accessing this attribute.
+
+        Returns
+        -------
+        WCS
+            An `~astropy.wcs.WCS` object representing the WCS of the widget.
+
+        """
+        if self._wcs == {}:
+            raise ValueError(
+                "You need to call synchronize_wcs before accessing the wcs attribute. "
+                "The WCS attribute need to be accessed from the next cell."
+            )
+        wcs = WCS(naxis=self._wcs["NAXIS"])
+        wcs.wcs.cdelt = [self._wcs["CDELT1"], self._wcs["CDELT2"]]
```

Comment:
can't we just loop through the dictionary keys/values retrieved from aladin.getViewWCS() ?
Would be more generic and less error prone.

Reply:
Looks like you can do

```python
from astropy.wcs import WCS
WCS({"CRVAL1": 0, "CRVAL2":1})
```

```
WCS Keywords

Number of WCS axes: 2
CTYPE : '' '' 
CRVAL : 0.0 1.0 
CRPIX : 0.0 0.0 
PC1_1 PC1_2  : 1.0 0.0 
PC2_1 PC2_2  : 0.0 1.0 
CDELT : 1.0 1.0 
NAXIS : 0  0
```

Reply:
oh yes, thanks, that's better!